plugins/qlcplus_4.11.2/import.js

```diff
@@ -155,33 +248,39 @@ function getOflCapability(qlcPlusCapability, qlcPlusChannel) {
       }
     },
     Colour() {
-      if (`Color` in qlcPlusCapability.$) {
-        cap.colors = [qlcPlusCapability.$.Color];
-
-        if (`Color2` in qlcPlusCapability.$) {
-          cap.colors.push(qlcPlusCapability.$.Color2);
-        }
-      }
+      if (channelName.match(/wheel\b/i)) {
+        cap.type = `WheelSlot`;
+        cap.slotNumber = qlcPlusChannel.Capability.indexOf(qlcPlusCapability) + 1;
 
-      if (channelName.match(/wheel/i)) {
-        cap.type = `ColorWheelIndex`;
         cap.comment = getSpeedGuessedComment();
 
         if (`speedStart` in cap) {
-          cap.type = `ColorWheelRotation`;
+          cap.type = `WheelRotation`;
+          delete cap.slotNumber;
         }
       }
       else {
         cap.type = `ColorPreset`;
         cap.comment = capabilityName;
+
+        if (`Color` in qlcPlusCapability.$) {
+          cap.colors = [qlcPlusCapability.$.Color];
+
+          if (`Color2` in qlcPlusCapability.$) {
+            cap.colors.push(qlcPlusCapability.$.Color2);
+          }
+        }
       }
     },
     Gobo() {
-      cap.type = `GoboIndex`;
+      cap.type = `WheelSlot`;
+      cap.slotNumber = qlcPlusChannel.Capability.indexOf(qlcPlusCapability) + 1;
+
       cap.comment = getSpeedGuessedComment();
 
       if (`speedStart` in cap) {
-        cap.type = `GoboWheelRotation`;
+        cap.type = `WheelRotation`;
+        delete cap.slotNumber;
```

Comment:
You could check if `fixture.wheels` includes the channel name. If not, set `type` to `WheelSlotRotation` and `wheel` to `Unknown`.

This especially makes sense with my suggestion on wheel creation (see line 97).